Prepare broker 20.10.6 / engine 20.10.6 / clib 20.10.3 / connectors 20.10.2 release notes (#862)
centreon/centreon-documentation: fr/releases/centreon-core.md

## Centreon Engine

### 20.10.6

> 

Reviewer: ```suggestion

`15 juillet 2021`

```

## Centreon Broker

### 20.10.6

> 

Reviewer: ```suggestion

`15 juillet 2021`

```

#### Correctifs

- Un *deadlock* pouvait se produire lorsque broker était arrêté immédiatement après son démarrage
- Un *core dump* pouvait apparaître lorsque centengine était stoppé
- Parfois centengine ne parvenait pas à se reconnecter au broker central
- Quand beaucoup de collecteurs essayent de se connecter au même moment au broker central, il arrive que les derniers ne puissent pas se connecter
- Les plages de temps n'étaient pas parsées correctement lorsqu'elles se terminaient par `\r` ou `\n`
- Problèmes de compilation sur RaspberryPi
- Les requêtes TLS n'étaient pas correctement traitées par GNUTLS sur RedHat 8

> 

Author: Same as in English version: it duplicates `Sometimes Engine cannot reconnect to the central broker when cbd is restarted
`

- Contient tous les correctifs jusqu'à la version 20.04.9

## Centreon CLib

## 20.10.3

### Bug fixes

- Libraries are loaded lazily now. This allows not to check all link issues during the load.

## 20.10.2

### Enhancement

> - Compilation in C++14 with conan-center: bintray has stopped. We had to switch to the conan-center. And then our conan dependencies had to upgrade and then we had to switch to C++14. So here is the corresponding compilation.

Reviewer 2: ```suggestion
- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to conan-center. And then our conan dependencies had to be upgraded and then we had to switch to C++14. So here is the corresponding compilation.
```

## 20.10.1

`February x, 2021`

### Bugfixes

> - Start/Stop process\_manager mechanism is rewritten and much more simpler. The consequences are that there are less dead locks in the processes management.

Reviewer 2: ```suggestion
- Start/Stop process\_manager mechanism is rewritten and much simpler. The consequences are that there are less deadlocks in the processes management.
```

- Contient tous les correctifs jusqu'à la version 20.04.9

## Centreon CLib

## 20.10.3

> 

Reviewer: ```suggestion

`15 juillet 2021`

```

### Bug fixes

- Libraries are loaded lazily now. This allows not having to check all link issues during loading.

## 20.10.2

`4 juin 2021`

### Enhancement

- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to conan-center. And then our conan dependencies had to be upgraded and then we had to switch to C++14. So here is the corresponding compilation.

## 20.10.1

`28 avril 2021`

### Bugfixes

- Start/Stop process\_manager mechanism is rewritten and much simpler. The consequences are that there are less deadlocks in the processes management.
- The simple cases that are addition, subtraction and some other cases, have had their computations simplified, that is to say less computations, so faster.

## 20.10.0

`October 21, 2020`

### Bugfixes

- A possible deadlock was removed when a process is added and at the same time a process is removed because in timeout.

## Centreon Connector Perl

### 20.10.2

> 

Reviewer: ```suggestion

`15 juillet 2021`

```

- Contient tous les correctifs jusqu'à la version 20.04.9

## Centreon CLib

> ## 20.10.3

Reviewer: ```suggestion
### 20.10.3
```

`15 juillet 2021`

> ### Bug fixes

Reviewer: ```suggestion
#### Bug fixes
```

- Libraries are loaded lazily now. This allows not having to check all link issues during loading.

> ## 20.10.2

Reviewer: ```suggestion
### 20.10.2
```

`4 juin 2021`

> ### Enhancement

Reviewer: ```suggestion
#### Enhancements
```

- Compilation in C++14 with conan-center: bintray has stopped. We had to switch to conan-center. And then our conan dependencies had to be upgraded and then we had to switch to C++14. So here is the corresponding compilation.

> ## 20.10.1

Reviewer: ```suggestion
### 20.10.1
```

`28 avril 2021`

> ### Bugfixes

Reviewer: ```suggestion
#### Bugfixes
```

- Start/Stop process\_manager mechanism is rewritten and much simpler. The consequences are that there are less deadlocks in the processes management.
- The simple cases that are addition, subtraction and some other cases, have had their computations simplified, that is to say less computations, so faster.

> ## 20.10.0

Reviewer: ```suggestion
### 20.10.0
```

`October 21, 2020`

> ### Bugfixes

Reviewer: ```suggestion
#### Bugfixes
```

## Centreon Connector SSH

### 20.10.2

> 

Reviewer: ```suggestion

`15 juillet 2021`

```
